### PR TITLE
Remove beta from Databricks doc

### PR DESCRIPTION
**Pull request by yunbodeng-db**
Remove beta from Databricks doc

### CODE REVIEW

**Comment by yunbodeng-db** [comment]
@microsoft-github-policy-service agree [company="Databricks"]

**Comment by yunbodeng-db** [comment]
@microsoft-github-policy-service agree
@microsoft-github-policy-service agree company="Databricks"

**Comment by Court72** [comment]
@bezhan-msft

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by bezhan-msft — CHANGES_REQUESTED** [review]

File: powerquery-docs/connectors/media/databricks/filtered-connectors.png
[the remark this answers is not in the document]
Comment (yunbodeng-db) [reply powerquery-docs/connectors/media/databricks/filtered-connectors.png:1]: Done. I replaced with pictures with higher resolutions/dimensions. PTAL.

**Comment by yunbodeng-db** [comment]
@DougKlopfenstein Ping Doug, can you take a look at the revision that I replaced with higher definition images? Thanks.

**Pull request merged**